app/scripts/services/get-toggles-for-release.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../../typings/tsd.d.ts" />
+/// <reference path="abstract-http-service.ts" />
+
+module labsFrontendApp
+{
+    export interface Toggle
+    {
+        id: number,
+        name: string
+        summary: string
+    }
+
+    export interface GetTogglesForRelease
+    {
+        execute( releaseId: number ): ng.IPromise<Array<Toggle>>
+    }
+
+    export class HttpGetTogglesForRelease extends HttpService<Array<Toggle>> implements GetTogglesForRelease
+    {
+        protected url = 'toggle/list';
```

Comment:
so the downside of testing them is that the only differences are GET params and the URL so you'd just be testing whether the URL was right and the params are mapped in okay